### PR TITLE
Add additional information to --version flag

### PR DESCRIPTION
**Pull request by YakDriver**

### CODE REVIEW

**Comment by YakDriver** [comment]
```console
$ watchmaker --version
Watchmaker/0.10.3 Python/3.6.5 Darwin/18.2.0
```

**Comment by YakDriver** [comment]
I flaked it up. I'll fix momentarily.

**Comment by lorengordon** [comment]
We also have a preference in this project for `.format()` over `%s` string formatting...

File: src/watchmaker/cli.py [hunk 1]
```diff
@@ -20,11 +20,18 @@
...
+def print_version(ctx, param, value):
+    if not value or ctx.resilient_parsing:
+        return
+    click.echo(watchmaker.VERSION_INFO)
+    ctx.exit()
 
 @click.command(context_settings=dict(
     ignore_unknown_options=True,
 ))
-@click.version_option(version=watchmaker.__version__)
```
[the remark this answers is not in the document]
Comment (lorengordon) [reply src/watchmaker/cli.py]: gotcha! 👍

**Comment by YakDriver** [comment]
go is all `%s` so I've forgotten the Pythonic ways. It's all fixed.

**Comment by lorengordon** [comment]
Looks good, Glgo ahead and merge if done!

**Pull request merged**